src/audits/manifest/short-name-length.js

```diff
@@ -48,16 +49,26 @@ class ManifestShortNameLength extends Audit {
   static audit(artifacts) {
     let isShortNameShortEnough = false;
     let debugString;
+    let resolvedName;
     const manifest = artifacts.manifest.value;
     const suggestedLength = 12;
 
-    if (manifest && manifest.short_name && manifest.short_name.value) {
+    if (manifest) {
+      const shortNameValue = _get(manifest, 'short_name.value');
+      const nameValue = _get(manifest, 'name.value');
+
+      // When no shortname can be found we look for a name
+      // https://developer.chrome.com/apps/manifest/name#short_name
+      resolvedName = shortNameValue || nameValue;
```

Comment:
Technically, an empty string for a `name` or a `short_name` (or a string of only whitespace, which is `trim`med down to the empty string) *is* a valid `name` or `short_name`, AFAICT [from the spec](https://www.w3.org/TR/appmanifest/#name-member). The current PR will fail a `resolvedName` of an empty string, so with that in mind should check against `undefined`.

*But*, I believe Chrome, at least, rejects empty strings for names, so it's worth still rejecting those. Testing for truthiness is an equivalent shorthand for `resolvedName !== undefined && resolvedName !== ''` since we know they're of type `string|undefined`. Probably still be worth making a comment that we're specifically rejecting the empty sting even though the web app manifest spec is cool with it, though.